install_config/upgrading/manual_upgrades.adoc

```diff
@@ -1370,6 +1370,26 @@ of Fluentd pods, the deployer does delete the *logging-fluentd* Daemonset and re
 it from the *logging-fluentd-template* template.
 ====
 
+The latest EFK stack now uses Elasticsearch 2.3 with a common data model. This
+means Fluentd sends logs to Elasticsearch with a new indexing pattern for
+projects. The pattern is:
```

Comment:
I think we should call out that the pattern for 'operations' logs will stay the same.